introduction.md

```diff
@@ -0,0 +1 @@
+Hi all. My name is Haleem ur Rasheed. Just started learning Github.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.